CommonTools/RecoAlgos/src/PrimaryVertexSorting.cc

```diff
@@ -27,7 +27,13 @@ float PrimaryVertexSorting::score(const reco::Vertex & pv,const  std::vector<con
     float scale=1.;
     if(c->bestTrack() != 0)
       {
-        scale=(c->pt()-c->bestTrack()->ptError())/c->pt();
+	if(c->pt()!=0) {
+       		 scale=(c->pt()-c->bestTrack()->ptError())/c->pt();
+	}
+ 	if(isnan(scale)) { 
```

Comment:
The static analyzer warns: "std::isnan does not work when fast-math is used. Please use edm::isNotFinite from 'FWCore/Utilities/interface/isNotFinite.h'".
Please fix this line.